### PR TITLE
refactor: remove the Feature reference from FeatureGeometry

### PR DESCRIPTION
**Pull request by zarov**
Even if it was a simple pointer, scenes with a lot of FeatureGeometry
were having a lot of memory taken because of those pointers.

### CODE REVIEW

File: src/Core/Feature.js [hunk 1]
```diff
@@ -77,49 +79,44 @@ export class FeatureGeometry {
...
+            coord.as(feature.crs, coord);
         }
-        if (this._feature.normals) {
-            coord.geodesicNormal.toArray(this._feature.normals, this._feature._pos);
+        if (feature.normals) {
+            coord.geodesicNormal.toArray(feature.normals, feature._pos);
         }
 
-        this._feature._pushValues(this._feature, coord.x, coord.y, coord.z);
+        feature._pushValues(feature, coord.x, coord.y, coord.z);
```
Comment (gchoqueux) [inline src/Core/Feature.js]: it's possible to remove `feature` from `_pushValues`  params?

**Review by gchoqueux — COMMENTED** [review]
add properties like: vertices, normal, crs 
and remove feature from function params.

File: src/Core/Feature.js (hunk 1 above)
Comment (zarov) [reply src/Core/Feature.js]: Done

**Comment by zarov** [comment]
> add properties like: vertices, normal, crs and remove feature from function params.

I'm not convinced by this, because in some case that would be a lot of parameters. [See this function](https://github.com/iTowns/itowns/blob/master/src/Core/Feature.js#L44-L52): we need the `crs`, `vertices` and then `normals` and `size` in the `_extendBuffer` call. WDYT ?

**Comment by gchoqueux** [comment]
see #1295

**Review by gchoqueux — APPROVED** [review]

**Pull request merged**